components/Common/Search/States/WithSearchBox.tsx

```diff
@@ -131,6 +183,21 @@ export const WithSearchBox: FC<SearchBoxProps> = ({ onClose }) => {
             <form onSubmit={onSubmit}>
               <input
                 ref={searchInputRef}
+                aria-activedescendant={
+                  selectedResult != undefined
+                    ? `search-hit-${selectedResult}`
+                    : undefined
+                }
+                aria-autocomplete="list"
+                aria-controls="fulltext-results-container"
+                aria-expanded={
```

Comment:
```
aria-expanded={Boolean(
  !searchError &&
  searchTerm &&
  searchResults && 
  searchResults.count
)}

Reply:
Changed it.

This is just to ensure the output right? 🤔